Mailgun_Subscriptions/Subscription_Form.php

```diff
@@ -135,7 +135,7 @@ protected function do_form_contents( $instance ) {
 			$user = wp_get_current_user();
 			$default_email = $user->user_email;
 		}
-		printf( '<input type="text" value="%s" name="mailgun-subscriber-email" size="20" id="mailgun-email-address-%d" />', $default_email, $instance_counter );
+		printf( '<input type="text" value="%s" name="mailgun-subscriber-email" size="20" id="mailgun-email-address-%d" required />', $default_email, $instance_counter );
```

Comment:
All modern browsers support the `required` attribute, and the email is required so...